tests/integration/test_vulnerability_detector/test_feeds/debian/test_extra_tags_debian_feed.py

```diff
@@ -0,0 +1,94 @@
+
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
```

Comment:
```suggestion
# Copyright (C) 2015-2020, Wazuh Inc.
# Created by Wazuh, Inc. <info@wazuh.com>.
```

Reply:
Done in ca26474b64f76ba719b585d27df0f1728233f213